servicetalk-grpc-netty/src/test/java/io/servicetalk/grpc/netty/ProtocolCompatibilityTest.java

```diff
@@ -716,8 +716,6 @@ private static void testRequestResponse(final CompatClient client, final TestSer
                 assertEquals(1000004, response4List.get(1).getSize());
                 assertEquals(1000005, response4List.get(2).getSize());
             }
-        } catch (Exception ex) {
-            ex.printStackTrace();
```

Comment:
Some tests were actually failing because of the `identity` check, but were marked as "passed" due to this `catch`:
```
Caused by: io.servicetalk.grpc.api.GrpcStatusException: UNIMPLEMENTED: Invalid grpc-encoding: identity
	at io.servicetalk.grpc.api.GrpcStatusException.of(GrpcStatusException.java:71)
	at io.servicetalk.grpc.api.GrpcUtils.readGrpcMessageEncodingRaw(GrpcUtils.java:357)
	at io.servicetalk.grpc.api.DefaultGrpcClientCallFactory.lambda$null$2(DefaultGrpcClientCallFactory.java:155)
	at io.servicetalk.concurrent.api.SingleFlatMapPublisher$SubscriberImpl.onSuccess(SingleFlatMapPublisher.java:76)
	at io.servicetalk.concurrent.api.ContextPreservingSingleSubscriber.onSuccess(ContextPreservingSingleSubscriber.java:75)
	at io.servicetalk.concurrent.api.RetryWhenSingle$RetrySubscriber.onSuccess(RetryWhenSingle.java:80)
	at io.servicetalk.concurrent.api.ContextPreservingSingleSubscriber.onSuccess(ContextPreservingSingleSubscriber.java:75)
	at io.servicetalk.concurrent.api.SingleFlatMapSingle$SubscriberImpl$1.onSuccess(SingleFlatMapSingle.java:88)
	at io.servicetalk.concurrent.api.ContextPreservingCancellableSingleSubscriber.onSuccess(ContextPreservingCancellableSingleSubscriber.java:42)
	at io.servicetalk.http.utils.BeforeFinallyHttpOperator$ResponseCompletionSubscriber.onSuccess(BeforeFinallyHttpOperator.java:132)
	at io.servicetalk.http.utils.BeforeFinallyHttpOperator$ResponseCompletionSubscriber.onSuccess(BeforeFinallyHttpOperator.java:96)
	at io.servicetalk.concurrent.api.ContextPreservingSingleSubscriber.onSuccess(ContextPreservingSingleSubscriber.java:75)
	at io.servicetalk.http.netty.H2ClientParentConnectionContext$DefaultH2ClientParentConnection$2.onSuccess(H2ClientParentConnectionContext.java:394)
	at io.servicetalk.http.netty.H2ClientParentConnectionContext$DefaultH2ClientParentConnection$2.onSuccess(H2ClientParentConnectionContext.java:386)
	at io.servicetalk.concurrent.api.ContextPreservingSingleSubscriber.onSuccess(ContextPreservingSingleSubscriber.java:75)
	at io.servicetalk.concurrent.api.MapSingle$1.onSuccess(MapSingle.java:51)
	at io.servicetalk.concurrent.internal.SubscriberUtils.safeOnSuccess(SubscriberUtils.java:302)
	at io.servicetalk.concurrent.api.TaskBasedAsyncSingleOperator$SingleSubscriberOffloadedTerminals.deliverTerminalToSubscriber(TaskBasedAsyncSingleOperator.java:118)
	at io.servicetalk.concurrent.api.TaskBasedAsyncCompletableOperator$AbstractOffloadedSingleValueSubscriber.deliverSignals(TaskBasedAsyncCompletableOperator.java:156)
	at io.servicetalk.concurrent.api.ContextPreservingRunnable.run(ContextPreservingRunnable.java:43)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```